### PR TITLE
docs: add React 18 setup to quick start docs

### LINKED ISSUE #30690: [Feature]: Update the setup fluent ui process as per React 18
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

With the shipment of react 18, React DOM's render has become deprecated (https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis) . So it would really be helpful if the documentation website for fluent ui 9 (https://react.fluentui.dev/?path=/docs/concepts-developer-quick-start--page) be updated as per new changes. 

### Have you discussed this feature with our team

_No response_

### Additional context

This is in the react Official Documentation
image

And this is the waring message generated in the google chrome's console: 
image

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

Normal

### PR DESCRIPTION
**Pull request by spmonahan**
## Previous Behavior

We didn't now show how to "quick start" Fluent v9 with [React 18 APIs](https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis).

## New Behavior

We show how to quick start Fluent v9 with React 18 APIs

## Related Issue(s)

- Fixes #30690

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9f929827e7b657907f84791494ffbd09b0b8ab3f'>9f929827e7b657907f84791494ffbd09b0b8ab3f</a></sub>
<div id="bundleSizeComment9423"/>

**Review by sopranopillow — APPROVED** [review]

File: apps/public-docsite-v9/src/Concepts/QuickStart.stories.mdx [hunk 1]
```diff
@@ -16,6 +16,26 @@ Fluent UI components are styled using CSS in JS. This technique requires a style
 
 Place a `<FluentProvider />` at the root of your app and pass theme as a prop.
 
+### React 18
+
+```jsx
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
```
Comment (tudorpopams) [inline apps/public-docsite-v9/src/Concepts/QuickStart.stories.mdx:24]: Do we want to use the `teamsLightTheme` here, or should we go with `webLightTheme` as default example?

Comment (spmonahan) [reply apps/public-docsite-v9/src/Concepts/QuickStart.stories.mdx:24]: We should change to `webLightTheme`. Looks like the Teams theme was there before so just got copy-pasted :)

**Pull request merged**